src/paid-cookie-consent/guidance-handler.js

```diff
@@ -56,7 +56,12 @@ export default async function handler(message, context) {
     url,
     guidanceParsed,
   );
-  await Suggestion.create(suggestionData);
+  const requiresValidation = Boolean(context.site?.requiresValidation);
+
+  await Suggestion.create({
+    ...suggestionData,
+    status: requiresValidation ? Suggestion.STATUSES.NOT_VALIDATED : Suggestion.STATUSES.NEW,
```

Comment:
To me, it looks that the cookie-consent is always marking the old opportunity entries as Ignored and creating new entries.

After your PR changes, would the paid-cookie-consent opportunity, even if existing, always be re-created with Not_Validated suggestion?

That looks like the wrong experience.